resources.md

```diff
@@ -115,6 +116,19 @@ to get started.
 
 </div>
 
+<div markdown="1" class="item">
+
+#### [Metabase](https://www.metabase.com/)
+
+Metabase is an open-source business intelligence platform. You can use  Metabase 
+to ask questions about your data, or embed Metabase in your app to let your 
+customers explore their data on their own. Project repository can be found 
+[here](https://github.com/starburstdata/metabase-driver). Follow the [Trino metabase-driver 
+instruction](https://github.com/starburstdata/metabase-driver#driver-plugin-installation)
```

Comment:
Maybe should also link to https://docs.starburst.io/data-consumer/clients/metabase.html

Reply:
link added